Xamarin.Forms.Core/GestureRecognizer.cs

```diff
@@ -1,9 +1,209 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
```

Comment:
We aren't fans of using Linq, can we remove this and do it without Linq ?

Reply:
done